op-node/rollup/driver/driver.go

```diff
@@ -181,6 +183,11 @@ func NewDriver(
 
 	opts := event.DefaultRegisterOpts()
 
+	if cfg.InteropTime != nil {
```

Comment:
this creates an interop driver even if interop time is in the future. That's probably fine, just checking that it's intended.

Reply:
It's intentional. The deriver module needs to be ready before interop activates. But there's no need to run it if the fork isn't scheduled yet.